tools/util/cassandra_stress.py

```diff
@@ -0,0 +1,50 @@
+from contextlib import contextmanager
+import subprocess
+
+class CassandraStress:
+    def __init__(self, cassandra_dir):
+        self.stress_cmd = f"{cassandra_dir}/tools/bin/cassandra-stress"
+
+
+    def run(self, command, workload_size, node_ip, keyspace="ks", threads=1):
+        subprocess.run(args=[self.stress_cmd, 
+                             command, "no-warmup", "n={}".format(workload_size), 
+                             "-node", node_ip, 
+                             "-rate", "threads={}".format(threads), 
+                             "-schema", "keyspace={}".format(keyspace)],
+                             check=True)
+
+
+@contextmanager
+def cassandra_stress(cassandra_version="3.11.16"):
+    try:
+        # Fetch the cassandra
+        print(f"Fetching cassandra {cassandra_version}")
+        cassandra_dir = f"apache-cassandra-{cassandra_version}"
+        cassandra_tar = f"{cassandra_dir}-bin.tar.gz"
+        # See https://www.apache.org/dyn/closer.lua/cassandra/3.11.16/apache-cassandra-3.11.16-bin.tar.gz
+        cassandra_url = f"https://dlcdn.apache.org/cassandra/{cassandra_version}/{cassandra_tar}"
+        subprocess.run(args=["wget", "-N", cassandra_url], check=True, stderr=subprocess.DEVNULL)
```

Comment:
Looking at the output of some example CI run (e.g. https://github.com/muzarski/cql-stress/actions/runs/6248180046/job/16962268279) I decided to drop the output and not display it, since it looks kind of messy. What do you think about that?

And why stderr? Well, I was confused as well but apparently `wget` prints everything to stderr instead of stdout.

Reply:
I think that you can use `-s` flag to silence this particular output. I'd rather not ignore stderr unconditionally in order not to skip some important messages (if any such messages can appear).

There is some other command lower here which redirects stdout - same concern there.